.circleci/run.sh

```diff
@@ -73,7 +73,7 @@ setup_repos()
         git checkout -f FETCH_HEAD
     fi
 
-    for proj in dmd druntime ; do
+    for proj in dmd druntime tools ; do
```

Comment:
The auto-tester and other CI behave like this already anyhow.

(Also locking tools to a specific version was done initially because it seemed as a good idea at the time, but in practice it just made updates more annoying.)